CHANGELOG.rst

```diff
@@ -13,6 +13,56 @@ Refactoring:
 * Don't mutate ``globals()`` in ``webargs.fields`` (:pr:`411`).
 * Use marshmallow 3's ``Schema.from_dict`` if available (:pr:`415`).
 
+* *Backwards-incompatible*: Schema fields may not specify a location any
+  longer, and `Parser.use_args` and `Parser.use_kwargs` now accept `location`
+  (singular) instead of `locations` plural. Instead of using a single field or
+  schema with multiple `locations`, users are recommended to make multiple
+  calls to `use_args` or `use_kwargs` with a distinct schema per location. For
+  example, code should be rewritten like this:
+
+.. code-block:: python
+
+    # under webargs v5
+    class CompoundSchema:
+        q1 = ma.fields.Int(location="query")
+        q2 = ma.fields.Int(location="query")
+        h1 = ma.fields.Int(location="headers")
+
+
+    @parser.use_args(CompoundSchema(), locations=("query", "headers"))
```

Comment:
Maybe use dicts instead of schema classes for these examples; I think that's the more common usage.

Reply:
Sure, I see now that other parts of the changelog use dicts. I'll make it consistent.